SofaKernel/SofaFramework/SofaMacros.cmake

```diff
@@ -705,6 +705,7 @@ function(sofa_set_install_relocatable target install_dir)
                 || true
             )
     endif()
+    set_target_properties(${target}_relocatable_install PROPERTIES FOLDER "relocatable_install")
```

Comment:
I'd put them next to their corresponding project with
```cmake
get_target_property(target_folder target FOLDER)
set_target_properties(${target}_relocatable_install PROPERTIES FOLDER ${target_folder})
```

Reply:
#prout!?

Reply:
We don't want to see relocatable_install projects so it's better to have them all in one folder and ignore it.